### PR TITLE
Ascci image

### PR DESCRIPTION
**Pull request by dhananjaylatkar**
ascci image rendering of camera output

### CODE REVIEW

File: src/CamReader.py [hunk 1]
```diff
@@ -44,6 +43,8 @@ def capture_image(self, w: int, h: int) -> np.ndarray:
         # if frame is read correctly ret is True
         if not ret:
             print("Can't receive frame (stream end?). Exiting ...")
+            self.close_camera()
+            exit(1)
```
Comment (garuna-m6) [inline src/CamReader.py:47]: Thanks this is better

File: src/ascci_utils.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+from asciimatics_mod import ColourImageFilePIL
+from CamReader import CamReader
+
+
+class ascci:
+    """Generic ascii methods"""
+
+    def __init__(self) -> None:
+        pass
```
Comment (garuna-m6) [inline src/ascci_utils.py:16]: won;t it be better if we assign screen  as a property of self ?

File: src/asciimatics_mod.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            ascii_image += "${%d} " % bg
+                    else:
+                        if fill_background or uni:
+                            ascii_image += "${%d,2,%d}%s" % (col2, col, brush)
+                        else:
+                            ascii_image += "${%d}#" % col
+            if uni:
+                ascii_image += "${%d,2,%d}." % (bg, bg)
+            self._images.append(ascii_image)
+
```
Comment (garuna-m6) [inline src/asciimatics_mod.py:93]: I am not certain about the exact implementations of PIL functions, but this seems to be taken from asciimatics lib directly, so lgtm

**Review by garuna-m6 — APPROVED** [review]
Mostly looks good to me, just need one explaination of the for loop in show_image function

File: src/asciimatics_mod.py (hunk 3 above)
Comment (dhananjaylatkar) [reply src/asciimatics_mod.py:93]: yes, it's taken from lib with little modifications.

File: src/ascci_utils.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        :param screen: The screen to use when displaying the image.
+        :param image: Color image from asccimatics.
+        :param position: The position of image on terminal.
+        """
+        effects = [
+            Print(screen, image, y=position[1], x=position[0], stop_frame=200),
+        ]
+        screen.set_scenes([Scene(effects, 0)])
+        try:
+            for _ in range(4):
```
[the remark this answers is not in the document]
Comment (dhananjaylatkar) [reply src/ascci_utils.py:31]: it needs min 4 iterations to show image on screen. not sure why.

File: src/ascci_utils.py (hunk 2 above)
Comment (dhananjaylatkar) [reply src/ascci_utils.py:16]: yes, if we decide to use ascci_utils for screen creation.

**Pull request merged**